app/scripts/app.js

```diff
@@ -38,6 +38,15 @@
             if (rejection.status === 401) {
               $injector.get('$state').go('login');
               $injector.get('$ionicLoading').hide();
+            }else if(rejection.status === 404){
+              var popupOptions = {
+               title: 'Error',
+               content: rejection.statusText
```

Comment:
esto te va a devolver Resource not found siempre... seguro lo quieres mostrar así?